SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -172,6 +174,7 @@ public final class SwiftMessageBar {
     @objc func didTapMessage(gesture: UITapGestureRecognizer) {
         let message = gesture.view as! Message
         dismissMessage(message, fromGesture: true)
+        self.tappedHandler()
```

Comment:
and then if it's optional, simply `tapHandler?()` (also omitting the `self.` as it's not needed)